css/selectors/after.json

```
@@ -5,6 +5,10 @@
         "__compat": {
           "description": "<code>::after</code>",
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::after",
+          "spec_url": [
+            "https://drafts.csswg.org/css-pseudo/#selectordef-after",
+            "https://drafts.csswg.org/selectors-3/#gen-content"
+          ],
```

Seems to me like the selectors level 3 definition is outdated.


```suggestion
          "spec_url": "https://drafts.csswg.org/css-pseudo/#generated-content",
```